DynamicTablesPkg/Include/ArchCommonNameSpaceObjects.h

```diff
@@ -779,6 +780,15 @@ typedef struct CmArchCommonPstateInfo {
   UINT32          Ppc;
 } CM_ARCH_COMMON_PSTATE_INFO;
 
+/** A structure that describes the _STA (Device Status) Info.
+
+    ID: EArchCommonObjStaInfo
+*/
+typedef struct CmArchCommonStaInfo {
+  /// Device Status
+  UINT32    DeviceStatus;

```

Comment:
Just a personnal note:
it would be good to have a way of validating the value in `DeviceStatus`, cf only bits 0-4 are valid and some bits are inter-dependent. But there is no generic way right now.